django_project/core/settings/base.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+"""
+core.settings.base
+"""
+# Django settings for projecta project.
+
+from .utils import absolute_path
+
+ADMINS = (
+    ('Dimas Ciputra', 'dimas@kartoza.com'),
```

Comment:
Could you add @timlinux (?) and myself here as well, please. I'd like to be getting emails etc for the first while.

Reply:
added